pkg/generate/app/cmd/newapp.go

```diff
@@ -315,7 +315,7 @@ func (c *AppConfig) componentsForRepos(repositories app.SourceRepositories) (app
 			df, err := info.Dockerfile.Dockerfile()
 			if err != nil {
 				// an unparseable docker file should not terminate the entire create - it may be valid to Docker but not to us
-				glog.V(1).Infof("The Dockerfile for the repository %q could not be parsed - no image stream can be created for the FROM", info.Path)
+				fmt.Printf("The Dockerfile for the repository %q could not be parsed - no image stream can be created for the FROM\n", info.Path)
```

Comment:
This should be output to stderr (the one provided to newapp) with a  "warning: " prefix